apis/accounts/index.yaml

```diff
@@ -0,0 +1,148 @@
+get:
+  tags:
+    - Accounts
+  operationId: "ListAccounts"
+  summary: "List accounts."
+  description: |
+    List all accounts known to and decrypted to this validator client
+  security:
+    - cookieAuth: []
+  parameters:
+    - name: page_size
+      in: query
+      required: false
+      description: "The maximum number of accounts to return in the response."
+      schema:
+        type: number
+    - name: page_token
+      in: query
+      required: false
+      description: "A pagination token returned from a previous call to `ListAccounts`. that indicates where this listing should continue from."
+      schema:
+        type: string
+  responses:
+    "200":
+      description: Success response
+      content:
+        application/json:
+          schema:
+            title: ListAccountsResponse
+            type: object
+            properties:
+              next_page_token:
+                type: string
+                description: "A pagination token returned from a previous call that indicates from where listing should continue."
+                example: "150"
+              total_size:
+                type: number
+                description: "Total count matching the request."
+              accounts:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    validating_pubkey:
+                      type: string
+                      description: "The validating public key."
+                    derivation_path:
+                      type: string
+                      description: "The derivation path (if present in the imported keystore)."
+                    slashing_protection_last_entry:
+                      type: string
+                      description: |
+                        Latest slashing protection DB data item associated with this pubkey.
+                        Signal no slashing protection DB data with a value of TBD (0, null, "")
+
+    "401":
+      $ref: "../../validator-oapi.yaml#/components/responses/Unauthorized"
+    "403":
+      $ref: "../../validator-oapi.yaml#/components/responses/Forbidden"
+    "500":
+      $ref: "../../validator-oapi.yaml#/components/responses/InternalError"
+
+post:
+  tags:
+    - Accounts
+  operationId: "ImportAccounts"
+  summary: "Import accounts."
+  description: |
+    Import keystores generated by the Eth2.0 deposit CLI tooling. All keystores MUST be encrypted with the same password.
+  security:
```

Comment:
Process of import keystores is CPU-heavy operation, so its very easy to put `validator_client` into busy state of procession (decrypting) keystores. So this request can be used to perform DoS attack on `validator_client`.
To resolve this issue we should specify maximum amount of keystores allowed for single call to `ImportAccounts`.

Reply:
I don't think any client restricts the number of keystores to be imported at once. Picking a number would be very arbitrary and restricting this a the spec level doesn't seem right. If a UI implementation wants to restrict it, it would be fine.